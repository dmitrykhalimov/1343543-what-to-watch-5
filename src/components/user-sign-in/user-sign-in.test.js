import React from "react";
import renderer from "react-test-renderer";
import UserSignIn from "./user-sign-in";
import {MemoryRouter} from 'react-router-dom';

describe(`UserSignIn`, () => {
  it(`Should UserSignIn render correctly`, () => {
    const tree = renderer
      .create(
          <MemoryRouter>
            <UserSignIn/>
          </MemoryRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
