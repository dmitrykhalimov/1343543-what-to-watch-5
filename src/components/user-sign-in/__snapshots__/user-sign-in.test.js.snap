// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserSignIn Should UserSignIn render correctly 1`] = `
<a
  className="user-block__link"
  href="/login"
  onClick={[Function]}
>
  Sign in
</a>
`;
